src/refmvs.rs

```diff
@@ -318,6 +318,8 @@ impl save_tmvs::Fn {
             if ri > rf.r.len() - R_PAD {
                 return ptr::null();
             }
+
+            const _: () = assert!(mem::size_of::<RefMvsBlock>() * (1 + R_PAD) > 16);
```

Comment:
This shouldn't be in this PR, should it?

Reply:
It should. There can't be anything in between the `unsafe` block and the `// SAFETY` comment, so I moved this assert to right before the safety comment.